### ISSUE TITLE
Cannot capture not invokable method error

### ISSUE THREAD

**Issue by falsandtru** [power-assert-js/power-assert#36]
```js
var o = {};
assert(o.m());
```

```
TypeError: assert._capt(...).m is not a function
```

```js
var o = {};
assert(assert._expr(assert._capt(assert._capt(o, 'arguments/0/callee/object').m(), 'arguments/0'), {
    content: 'assert(o.m())',
    ...
}));
```

**Comment by twada** [comment]
Thank you for reporting!

It's hard to fix...

I tried enclosing capture logic in function calls.
It seem to be a solution for me, but it does not work since it changes contextual semantics and dynamics around asynchronous nature, such as `async/await`.
I learned that I should not enclose original calls in extra function.

So, it's a `wontfix` issue for now and I should note this in FAQ section. Thanks!
